### DISCUSSION TITLE
Reversion to HDG mode

### DISCUSSION THREAD

**Discussion by cadilhac** [flybywiresim/aircraft#3628] [Q&A] [answered]
Am I right in saying that the following is simply not implemented yet and that it will be worked out in the Custom FBW build?

When I takeoff and have no flight plan (I simply enter the departure airport and get my vspeeds), on initial climb I should hear a triple click and NAV should revert to HDG mode because it simply doesn't know what to do? This is how I saw it work in the Airline2Sim course.
So right now, NAV stays active and as soon as I activate autopilot, the plane starts turning left indefinitely...

**Comment by hotshotp** [comment, marked as answer]
Roger that, it is not implemented. I will relay this to the person that is creating our autopilot! Thank you for this info.

**Reply by cadilhac** [reply]
Thanks. Let me add that it should do this anytime the status is NAV and there is nothing known anymore (after a crossed waypoint in a flight plan if it's the last one for instance). So, triple click and HDG mode engages (green + box).

**Reply by cadilhac** [reply]
Addendum: Trying to force NAV again should wait a few seconds (it tries to determine what to do) before HDG reengages again.
